misk/web/tabs/web-actions/src/rewrite/WebActionCardBody.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useCallback } from "react"
+import { Menu, H5, UL } from "@blueprintjs/core"
+import { WebActionMetadata } from "./types"
+import WebActionCollapse from "./WebActionCollapse"
+
+interface Props {
+  webActionMetadata: WebActionMetadata
+}
+
+export default function WebActionCardBody({ webActionMetadata }: Props) {
+  const allowedServicesTitle = useCallback(
```

Comment:
We can remove useCallback here as well. I think this could also be genericised as:
```
const renderTitle = (label, contents) => {
  return [label, contents ? `(${contents.join(', ')})` : null]
    .filter(value => !!value)
    .join(' ')
}
```